README.md

```diff
@@ -106,7 +106,7 @@ does all the setup for you. Clone that repo and make sure you're either on Broad
 to the VPN. Then run the following command:
 
 ```shell
-./db/psql-connect.sh dev tsps
+./db/psql-connect.sh dev teaspoons
```

Comment:
does this work already? any changes we need to make in that script?

Reply:
it does not, we'd have to make all the related tsps -> teaspoon changes affter this pr gets merged